__test__/users.test.js

```diff
@@ -123,31 +123,37 @@ describe('USER PASSWORD RESET', () => {
 });
 
 describe('GET /users', () => {
-  it('should return ', () => {
+  it('should return a 201 code', async () => {
+    const user = {
+      id: 1,
+      username: 'john'
+    };
+    const jwtToken = await jwt.generateToken(user);
     return request(server)
-      .get('/api/user/all')
+      .get('/api/user/john/users')
+      .set({ token: jwtToken })
       .then(res => {
-        expect(res.status).toBe(500);
+        expect(res.status).toBe(201);
```

Comment:
status code 201 is when you created a new Item...
kindly use 200

Reply:
Changed the status codes now